helix-term/src/ui/editor.rs

```diff
@@ -569,9 +572,22 @@ impl EditorView {
         cxt: &mut commands::Context,
         event: KeyEvent,
     ) -> Option<KeymapResult> {
+        self.autoinfo = None;
         match self.keymaps.get_mut(&mode).unwrap().get(event) {
             KeymapResult::Matched(command) => command.execute(cxt),
-            KeymapResult::Pending(_map) => { /* render infobox */ }
+            KeymapResult::Pending(map) => {
+                let mut body = BTreeMap::new();
```

Comment:
Why `BTreeMap` instead of `HashMap`?

Reply:
Hashmaps don't have an inherent order, so every time the infobox is shown it will have the items ordered differently.